server/configmodule.py

```diff
@@ -11,10 +11,6 @@ class Config:
     WEBDRIVER = False
     LOCAL = False
     LITE = False
-    # Use StatVarObs instead of PopObs data model.
-    # TODO(shifucun): change this to True after setting svobs_mode=true in mixer
-    # by default.
-    SVOBS = False
```

Comment:
Should it be set to True in the default config? There are still a few more references to this var around (e.g. kg_main.html). 

Let's flip it to True by default first, and clean up by deleting it in a follow up (similar to the protocol for an experiment launch).

Reply:
Good catch. Set it to True now